versions.yaml

```diff
@@ -154,8 +154,8 @@ assets:
         name: *glibc-initrd-name
         version: *glibc-initrd-version
       x86_64:
-        name: *default-initrd-name
-        version: *default-initrd-version
+        name: *glibc-initrd-name
+        version: *glibc-initrd-version
```

Comment:
Hmm.  Out of curiosity, why are we doing this change here?  It's not clear from the commit message.

Reply:
Based on my understanding, default-initrd-name generates an image for Alpine and we need image for Ubuntu 20.04 for our test.

Reply:
Hmm.  We have a problem here.  By the moment we do this, we are doing an unexpected change for all the Kata Containers consumers, not only the ones who will be using SEV.

We should consider adding a `sev`, or `tee` section here in case this is re-used by others, and avoid changing the default of what users consume.

Reply:
IIUC it is not changing the default, but rather instructing the build scripts to build the initrd-sev (I suppose it will have the -sev suffix) from Ubuntu 20.04 just like for ppc64le.

Let me build it in my local host to double-check that.

Reply:
@wainersm I just forced push to my branch. So now it has "sev" in versions.yaml
So the previous change was changing all x86_64 initrd builds from default-initrd-name to glib-initrd-name.
So I explicitly changed it only for SEV case now.

Reply:
And the regular, non-sev, x86_64 initrd is still based on Alpine (./opt/kata/share/kata-containers/kata-alpine-3.15.initrd)